Modules/CaPTkSurvival/include/CaPTkSurvival.h

```diff
@@ -0,0 +1,94 @@
+#ifndef CaPTkSurvival_h
+#define CaPTkSurvival_h
+
+// The following header file is generated by CMake and thus it's located in
+// the build directory. It provides an export macro for classes and functions
+// that you want to be part of the public interface of your module.
+#include <MitkCaPTkSurvivalExports.h>
+
+#include <QObject>
+#include <QFuture>
+#include <QFutureWatcher>
+#include <QProgressBar>
+
+namespace captk {
+/** \class CaPTkSurvival
+ *  \brief Survival Prediction Index Module API
+ */
+class MITKCAPTKSURVIVAL_EXPORT CaPTkSurvival : public QObject
+{
+    Q_OBJECT
+
+public:
+    CaPTkSurvival(QObject *parent = 0);
+
+    ~CaPTkSurvival() {}
+    QString cbicaModelDir;
+
+    /** \brief Runs the algorithm
+     * 
+     * Execute the algorithm in a background thread. When the
+     * algorithm finishes, OnAlgorithmFinished() is called.
+     * 
+     * @param QString subjectDirPath path to the subject directory
+     * @param QString modelDirPath path to the model directory
+     * @param QString outputDirPath path to the desired output directory
+     * @param bool trainNewModel whether to train a new model or not (if false, use an existing model)
+     * @param bool useCustomModel whether a custom model is being used (if false, use CBICA CaPTk model)
+    */
+    void Run(
+            QString modelDir,
+            QString subjectDir,
+            QString outputDir,
+            bool trainNewModel,
+            bool useCustomModel
+    );
+
+    void SetProgressBar(QProgressBar* progressBar);
```

Comment:
Remove progress bar stuff (and remove Qt|Widgets from module dependencies)

Reply:
Done, will update this PR momentarily